README.md

```diff
@@ -54,19 +58,21 @@ export default App;
 - `sketch`: This is the sketch script which should be executed in the p5 canvas.
 - You can also add as many custom properties as you want.
 
-In the below example you see the `myCustomRedrawAccordingToNewPropsHandler` function, which is called when the properties of a wrapper component are changed.
+In the below example you see the `myCustomRedrawAccordingToNewPropsHandler`
+function, which is called when the properties of a wrapper component are
+changed.
 
 ```js
-import React from "react";
-import P5Wrapper from "react-p5-wrapper";
+import React from 'react';
```

Comment:
What is the reason to change the formatting again after we agreed on double quotes months ago? 

I'm wondering why you're redoing everything I've already done. There is no improvement or value here. Perhaps, I'm missing something.

Reply:
So the quotes we could use doubles but prettier defaults on most other file types to singles and so for consistency with other file types I figured it would be better to stick with that, WDYT?

Reply:
https://github.com/jamesrweb/react-p5-wrapper/pull/79/commits/ca0655101544954839df03e85945284bdcff937b

Reply:
If you really want double quotes then I can set it back, it makes no difference at the end of the day so it's up to you?

Reply:
Updated 🤷🏻‍♂️